.github/workflows/release.yml

```diff
@@ -52,14 +52,17 @@ jobs:
         run: |
           export PATH=${PATH}:`go env GOPATH`/bin
           go get -v -t -d ./...
-      - name: Install gox
+      - name: Install bingo and deps
         run: |
           export PATH=${PATH}:`go env GOPATH`/bin
-          go get github.com/mitchellh/gox  
+          go get github.com/bwplotka/bingo && go mod tidy
+          bingo get
       - name: Compile
         run: |
+          VERSION=${GITHUB_REF##*/}
           export PATH=${PATH}:`go env GOPATH`/bin
-          gox -osarch="linux/amd64 linux/386 linux/arm darwin/amd64 windows/amd64" -output="pow-{{.OS}}-{{.Arch}}" ./cmd/pow
+          source .bingo/variables.env
+          $(GOX) -osarch="linux/amd64 linux/386 linux/arm darwin/amd64 windows/amd64" -output="pow-{{.OS}}-{{.Arch}}" -ldflags="$(GOVVV) -flags -version $(VERSION) -pkg $(go list ./buildinfo)" ./cmd/pow
```

Comment:
Using bingo provided env vars here to execute gox and govvv. Here, we pass the release/tag version to govvv so it uses it to set the build info `version`.